### PR TITLE
Docs 2858: Document Klarna Disputes app

### PR DESCRIPTION
**Pull request by FrancescaNCook**
Do the changes meet MultiSafepay Docs' Definition of Done?
- Style guide, tone of voice, and naming conventions
- Tested
- Reviewed on mobile, tablet and desktop screens
- Notified stakeholders
- Pass GitHub Actions

### CODE REVIEW

File: content/payments/methods/pay-later/klarna/user-guide/handling-disputes.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+meta_title: "Handling Klarna disputes - MultiSafepay Docs"
+read_more: "."
+weight: 1
+url: '/klarna/handling-disputes/'
+---
+Klarna provides support for resolving order disputes with customers. For a step-by-step overview, see Klarna – [How to handle disputes at Klarna](https://docs.klarna.com/disputes/).
+
+The easiest way to handle disputes is using the Disputes app in the Klarna Merchant Portal. The portal also gives you an overview of all your orders. 
+
+{{< blue-notice >}} **Note:** If viewing orders in the portal, you must still update [order and transaction statuses](/payment-methods/klarna/payment-flow/) manually in your MultiSafepay account or automatically via our API. {{< /blue-notice >}}
```
Comment (Sarah-MultiSafepay) [inline content/payments/methods/pay-later/klarna/user-guide/handling-disputes.md:12]:
```suggestion
{{< blue-notice >}} **Note:** If viewing orders in the Klarna Merchant Portal, you must still update [order and transaction statuses](/payment-methods/klarna/payment-flow/) manually in your MultiSafepay account or automatically via our API. {{< /blue-notice >}}
```

File: content/payments/methods/pay-later/klarna/user-guide/handling-disputes.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+6. To sign in for the first time, enter your email address and the password you just created, and then click **Log in**.
+
+### Setting up two-factor authentication
+
+If you want to increase the security of your account, set up two-factor authentication with the Google Authenticator app.
+
+1. Under **To-dos**, click **Activate two-factor authentication**. 
+2. In the dialog, click **Start the setup**.
+3. Open the Google Authenticator app and scan the QR code. 
+4. Enter the one-time code from the app, and click **Set up authentication**.
```
Comment (Sarah-MultiSafepay) [inline content/payments/methods/pay-later/klarna/user-guide/handling-disputes.md:31]:
```suggestion
4. Enter the one-time authorization code from the app, and click **Set up authentication**.
```

File: content/payments/methods/pay-later/klarna/user-guide/handling-disputes.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+## Getting started with Disputes
+
+To open the Disputes app, in the lefthand side menu, click **Disputes**.
+
+The side menu contains three pre-set filters to view:
+
+- All disputes
+- Unauthorized disputes
+- Diputes expiring soon
```
Comment (Sarah-MultiSafepay) [inline content/payments/methods/pay-later/klarna/user-guide/handling-disputes.md:41]:
```suggestion
- Disputes expiring soon
```

File: content/payments/methods/pay-later/klarna/user-guide/handling-disputes.md (hunk 1 above)
Comment (Sarah-MultiSafepay) [reply content/payments/methods/pay-later/klarna/user-guide/handling-disputes.md:12]: @FrancescaNCook, how are the orders automatically updated? Will Klarna inform MultiSafepay to update and then the merchant will get a notification?

**Review by Sarah-MultiSafepay — COMMENTED** [review]
I've made a couple of suggestions, raised one clarification question and corrected one typo.

File: content/payments/methods/pay-later/klarna/user-guide/handling-disputes.md (hunk 1 above)
Comment (FrancescaNCook) [reply content/payments/methods/pay-later/klarna/user-guide/handling-disputes.md:12]: @Sarah-MultiSafepay  Good question, but it just means "as normal in your integration". I'll try to rephrase.

Comment (FrancescaNCook) [reply content/payments/methods/pay-later/klarna/user-guide/handling-disputes.md:12]:
```suggestion
{{< blue-notice >}} **Note:** If viewing orders in the portal, remember you must still [change the order status to shipped](/payment-methods/klarna/changing-order-status-to-shipped/) as normal. {{< /blue-notice >}}
```

**Pull request merged**